### PR TITLE
Dont add SLE_ prefix to repo in openQA if it starts with SL

### PR DESCRIPTION
**Pull request by andrii-suse**
This will break the plugin MR, so they will need to regenerate the projects

### CODE REVIEW

**Comment by andrii-suse** [comment]
see https://gitlab.suse.de/openqa/openqa-trigger-from-ibs-plugin/-/merge_requests/206/diffs

File: script/scriptgen.py [hunk 1]
```diff
@@ -1089,7 +1089,12 @@ def gen_print_openqa(self, f):
         if self.repos or self.repolink:
             if self.ag.brand == "ibs":
                 self.p(" i=9", f)  # temporary to simplify diff with old rsync scripts, may be removed later
-            self.p(cfg.openqa_call_repot(self.build_id_from_iso, self.repos), f, "REPOTYPE", "''", "REPOPREFIX", "SLE_")
+            # some trickery for REPO_SLE_ vs REPO_SL_ variables in ibs
+            sl = "SLE_"
+            for r in self.repos:
+                if r.tag.startswith("SL"):
+                    sl = ""
```
Comment (Martchus) [inline script/scriptgen.py:1096]: Maybe a break would make sense here as well, or any like in this example: https://stackoverflow.com/questions/2012611/any-function-in-python-with-a-callback

Comment (andrii-suse) [reply script/scriptgen.py:1096]: sure, it was a test for whether anybody tries to follow this madness :)

**Review by jlausuch — APPROVED** [review]
LGTM

**Pull request merged**